### PR TITLE
Feature/profile page for mvp

### PR DESCRIPTION
**Pull request by BamfordJ**
I have updated the profile page with a new profile card, and have added real data about users requests and services. I have also modified the update profile page styling and allowed it to display the users current information ready for them to edit. The photo still needs to be sorted - I didn't attempt to look at this.

### CODE REVIEW

File: reachout/src/main/webapp/WEB-INF/jsp/updateProfile.jsp [hunk 1]
```diff
@@ -11,81 +11,93 @@
...
+								</div>
+								
+								<!-- Edit Health Status -->
+								<div class="form-group">
+									<label for="healthStatus">Status</label>
+									<select id="healthStatus" name="healthStatus" class="form-control">
 										<c:forEach var="hs" items="${healthList}">
-											<option><c:set var="h" value="${hs}" />
-												<c:out value="${h }" /></option>
+											<option ${hs == healthStatus ? 'selected' : ''}><c:set var="h" value="${hs}" />
```
Comment (BellJohn) [inline reachout/src/main/webapp/WEB-INF/jsp/updateProfile.jsp:56]: Nice. I've been trying to find a neat way to do this kind of logic on the front end

File: reachout/src/main/webapp/WEB-INF/jsp/updateProfile.jsp [hunk 2]
```diff
@@ -11,81 +11,93 @@
...
+					</div>
 				</div>
-			</div>
 
-		</div>
 
-		<c:choose>
-			<c:when test="${empty errors}">
+				<c:choose>
+					<c:when test="${empty errors}">
```
Comment (BellJohn) [inline reachout/src/main/webapp/WEB-INF/jsp/updateProfile.jsp:71]: Can condense that down to 
`<c:when test="${not empty errors}">
Errors : ${errors}
</c:when
`

But it's super minor so I wouldn't worry.

Comment (BamfordJ) [reply reachout/src/main/webapp/WEB-INF/jsp/updateProfile.jsp]: Updated it anyway :-)

File: reachout/src/main/webapp/WEB-INF/jsp/updateProfile.jsp [hunk 3]
```diff
@@ -66,14 +66,7 @@
...
 
-
-				<c:choose>
-					<c:when test="${empty errors}">
-					</c:when>
-					<c:otherwise>
-						Errors :  ${errors}
-					</c:otherwise>
-				</c:choose>
+				<c:when test="${not empty errors}"> Errors : ${errors} </c:when>
```
Comment (BellJohn) [inline reachout/src/main/webapp/WEB-INF/jsp/updateProfile.jsp:69]: Does it work when not wrapped in the <c:choose> tags? Front end isn't my forte so I'm not certain

Comment (BamfordJ) [reply reachout/src/main/webapp/WEB-INF/jsp/updateProfile.jsp]: No! It doesn't work without the choose tags. Perhaps I should have spent a second testing that change before committing it. Thanks

**Pull request merged**